app/Controller/PagesController.php

```diff
@@ -12,151 +12,170 @@
  *
  * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
  * @link          http://cakephp.org CakePHP(tm) Project
- * @package       app.Controller
  * @since         CakePHP(tm) v 0.2.9
+ *
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
-
 App::uses('AppController', 'Controller');
 
 /**
- * Static content controller
+ * Static content controller.
  *
  * Override this controller by placing a copy in controllers directory of an application
  *
- * @package       app.Controller
  * @link http://book.cakephp.org/2.0/en/controllers/pages-controller.html
  */
-class PagesController extends AppController {
+class PagesController extends AppController
+{
+    /**
+     * This controller does not use a model.
+     *
+     * @var array
+     */
+    public $uses = array();
+    public $uses2 = array();
+    public $uses3 = array();
 
 /**
- * This controller does not use a model
- *
- * @var array
+ * TooManyMethods.
  */
-	public $uses = array();
-	public $uses2 = array();
-	public $uses3 = array();
-
-/**
- * TooManyMethods
-*/
-public function display1() {
-}
-public function display2() {
-}
-public function display3() {
-}
-public function display4() {
-}
-public function display5() {
-}
-public function display6() {
-}
-public function display7() {
-}
-public function display8() {
-}
-public function display9() {
-}
-public function display10() {
-}
-public function display11() {
-}
-public function display12() {
-}
-public function display13() {
-}
-public function display14() {
-}
-public function display15() {
-}
-public function display16() {
-}
-public function display17() {
+public function display1()
+{
 }
+    public function display2()
+    {
+    }
+    public function display3()
+    {
+    }
+    public function display4()
+    {
+    }
+    public function display5()
+    {
+    }
+    public function display6()
+    {
+    }
+    public function display7()
+    {
+    }
+    public function display8()
+    {
+    }
+    public function display9()
+    {
+    }
+    public function display10()
+    {
+    }
+    public function display11()
+    {
+    }
+    public function display12()
+    {
+    }
+    public function display13()
+    {
+    }
+    public function display14()
+    {
+    }
+    public function display15()
+    {
+    }
+    public function display16()
+    {
+    }
+    public function display17()
+    {
+    }
 
-/**
- * Displays a view
- *
- * @return void
- * @throws NotFoundException When the view file could not be found
- *	or MissingViewException in debug mode.
- */
-	public function display() {
-    $eeeeeeeeee;
-    $aaaaaaaaaa;
-    $bbbbbbbbbb;
-    $aaaaaaaaaa;
+    /**
+     * Displays a view.
+     *
+     * @throws NotFoundException When the view file could not be found
+     *                           or MissingViewException in debug mode.
+     */
+    public function display()
```

Comment:
__[staging][phpmd]__
* The variable $title_for_layout is not named in camelCase.
* The variable $title_for_layout is not named in camelCase.